src/Routes/RoutesCommon.tsx

```diff
@@ -269,4 +284,56 @@ export const routes: Route[] = [
       window.location.hostname !== getConfig().VITE_LOCAL_IP,
     main: Flags,
   },
+  {
+    path: '/limit',
+    label: '',
+    icon: <SwapIcon />,
+    hideDesktop: true,
+    main: Trade,
+    category: RouteCategory.Featured,
```

Comment:
q: Do we really need all those icons etc on hidden routes?

Reply:
Not `category` and `icon` indeed, nice catch!  14f8744a6c